### DISCUSSION TITLE
Support custom command to deploy

### DISCUSSION THREAD

**Discussion by callmewhy** [Dokploy/dokploy#442] [Ideas]
Hi,

I'd like to run migrate before running the service. In GitHub action I do like this:

```sh
docker compose -f docker-compose.yml build
docker compose -f docker-compose.yml run --rm sega-api npx dotenv-cli -e .env.production -- prisma migrate deploy
docker compose -f docker-compose.yml up -d
```

But in dokploy, it's deployed with predefined docker compose commands, which is not allowed to modify(append only which is not enough as the migration need to be executed before running the service)

Do you have a plan for custom command deployment?

Thanks for your awesome project!

**Comment by fordnox** [comment]
a solution could be to use another docker container to run migration, and use depends_on
